engine/flink/management/periodic/src/main/scala/pl/touk/nussknacker/engine/management/periodic/AdditionalDictComponentConfigsExtractor.scala

```diff
@@ -0,0 +1,28 @@
+package pl.touk.nussknacker.engine.management.periodic
+
+import pl.touk.nussknacker.engine.api.component.{ComponentAdditionalConfig, DesignerWideComponentId}
+import pl.touk.nussknacker.engine.api.parameter.ValueInputWithDictEditor
+
+object AdditionalDictComponentConfigsExtractor {
+
+  // This function filters additional configs provided by AdditionalUIConfigProvider
+  // to include only component and parameter configs with Dictionary editors.
+  // This is done to reduce data sent to Flink as only configs regarding this editor are required to be known during execution.
+  def getAdditionalConfigsWithDictParametersEditors(
```

Comment:
let's think about a better name. I'd rather focus on the fact that these additional UI configs are needed in the runtime. Not on the current case which is the dict param editor.

Reply:
I changed the class name to `AdditionalComponentConfigsForRuntimeExtractor` and function's name to `getRequiredAdditionalConfigsForRuntime` not to tie it too much to dict param editors